frontend/awx/common/awx-toolbar-filters.tsx

```diff
@@ -77,7 +77,7 @@ export function useCreatedByToolbarFilter() {
   return useMemo<IToolbarFilter>(
     () => ({
       key: 'created-by',
-      label: t('Created by'),
+      label: t('Created'),
```

Comment:
I think this needs to stay as `Created by` cause `Created` is another filter. One get's a user the other gets the date the resource was created